conda/recipes/librmm/meta.yaml

```diff
@@ -51,8 +51,16 @@ outputs:
     requirements:
       build:
         - cmake {{ cmake_version }}
+        # We require spdlog and fmt (which was devendored from spdlog
+        # conda-forge packages in 1.11.0) so that the spdlog headers are not
+        # pulled by CPM and installed as a part of the rmm packages. However,
+        # building against librmm still requires these headers so they are also
+        # exported as a run requirement.
+        - fmt {{ fmt_version }}
+        - spdlog {{ spdlog_version }}
```

Comment:
Can these be moved to a `host` section instead of the `build` section?

Reply:
Edited in e5272cb9dd350c190679c4e02e3a1e7f8138efdc.